pkg/storage/driver/sql.go

```diff
@@ -320,10 +324,6 @@ func (s *SQL) Query(labels map[string]string) ([]*rspb.Release, error) {
 		releases = append(releases, release)
 	}
 
-	if len(releases) == 0 {
-		return nil, ErrReleaseNotFound
-	}
-
```

Comment:
It might be better to leave this for now, so as not to change the behaviour of an error being raised when there were releases but they couldn't be parsed (it might make it easier / clearer to get a subsequent PR approved which updates this to a different error).

Reply:
I agree. I shouldn't have break the current behaviour. @hickeyma what do you think? Thanks.

Reply:
Yes. We cannot change existing behavior unless it's unintended behavior (bug) or a security issue.

Reply:
@bacongobbler I agree. I have reverted that now. This PR now only contains the fix of return ErrReleaseNotFound immediately when no records are found so that it's consistent with the other drivers.